hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -1631,6 +1633,7 @@ public ReplicaHandler createRbw(
       if (ref == null) {
         ref = volumes.getNextVolume(storageType, storageId, b.getNumBytes());
       }
+      LOG.info("Creating Rbw, block: {} on volume: {}", b, ref.getVolume());
```

Comment:
is this really necessary? IMO logging one message for every rbw is just too much.

Reply:
> is this really necessary? IMO logging one message for every rbw is just too much.

I will change this to DEBUG level, do you think it is OK?